CNV-37943: Mounting the Windows driver disk on an existing VM (#78364)
openshift/openshift-docs: modules/virt-creating-vm-instancetype.adoc

ifdef::dynamic-key[]
. Set *Dynamic SSH key injection* in the *VirtualMachine details* section to on.
endif::[]

> . Optional: If you are creating a Windows VM, you can mount a Windows driver disk:
> .. Click the *Customize VirtualMachine* button.
> .. On the *VirtualMachine details* page, click *Storage*.
> .. Select the *Mount Windows drivers disk* checkbox.

Reviewer: ```suggestion
[TIP]
====
To create a Windows VM, you can go to the *Customize VirtualMachine* -> *Storage* -> *Mount Windows drivers disk*.
====
```

> .. Select the *Mount Windows drivers disk* checkbox.

Reviewer 2: Consider prepending "MicroSoft" Windows, if MSoft do own this.
